### PR TITLE
Joomla 4 Compatability

### PR DESCRIPTION
**Pull request by ankush-maherwal**

### CODE REVIEW

File: src/com_tjnotifications/site/views/preferences/tmpl/default_bs3.php [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+?>
+
+<form action="index.php?option=com_tjnotifications&view=preferences" method="post" id="adminForm" name="adminForm">
+	<div class="row-fluid">
+		<div class="span6">
+		</div>
+	</div>
+
+	<div id="display_info"></div>
+		<ul class="nav nav-tabs">
```
Comment (manojLondhe) [inline src/com_tjnotifications/site/views/preferences/tmpl/default_bs3.php:122]: fix indent in all layout files

File: src/com_tjnotifications/site/views/preferences/tmpl/default_bs3.php [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+					href="<?php echo('#'.$menu->client); ?>">
+					<?php echo str_replace("com_","",$menu->client); ?>
+				</a>
+			</li>
+		<?php endforeach; ?>
+		<?php endif; ?>
+		</ul>
+
+	<div class="tab-content">
+	<?php foreach ($this->clients as $i => $menu) :?>
```
Comment (manojLondhe) [inline src/com_tjnotifications/site/views/preferences/tmpl/default_bs3.php:136]: fix indent in all layout files

File: src/com_tjnotifications/site/views/preferences/tmpl/default_bs5.php [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+?>
+
+<form action="index.php?option=com_tjnotifications&view=preferences" method="post" id="adminForm" name="adminForm">
+	<div class="row">
+		<div class="col-xs-12 col-md-6">
+		</div>
+	</div>
+
+	<div id="display_info"></div>
+		<ul class="nav nav-pills" id="myTab" role="tablist">
```
Comment (manojLondhe) [inline src/com_tjnotifications/site/views/preferences/tmpl/default_bs5.php:122]: fix indent in all layout files

**Review by manojLondhe — CHANGES_REQUESTED** [review]
see comments

File: src/com_tjnotifications/site/views/preferences/tmpl/default_bs3.php (hunk 1 above)
Comment (ankush-maherwal) [reply src/com_tjnotifications/site/views/preferences/tmpl/default_bs3.php]: done

File: src/com_tjnotifications/site/views/preferences/tmpl/default_bs5.php (hunk 3 above)
Comment (ankush-maherwal) [reply src/com_tjnotifications/site/views/preferences/tmpl/default_bs5.php]: done

File: src/com_tjnotifications/site/views/preferences/tmpl/default_bs3.php (hunk 2 above)
Comment (ankush-maherwal) [reply src/com_tjnotifications/site/views/preferences/tmpl/default_bs3.php]: done

File: src/plugins/user/tjnotificationsmobilenumber/tjnotificationsmobilenumber.xml [hunk 4]
```diff
@@ -5,10 +5,10 @@
 	<author>Techjoomla</author>
 	<authorEmail>extensions@techjoomla.com</authorEmail>
 	<authorUrl>https://techjoomla.com</authorUrl>
-	<creationDate>22nd Jul 2021</creationDate>
-	<copyright>Copyright (C) 2016 - 2021 Techjoomla. All rights reserved.</copyright>
+	<creationDate>25th Jan 2022</creationDate>
+	<copyright>Copyright (C) 2016 - 2022 Techjoomla. All rights reserved.</copyright>
 	<license>http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL</license>
-	<version>2.0.1</version>
+	<version>3.0.0</version>
```
Comment (manojLondhe) [inline src/plugins/user/tjnotificationsmobilenumber/tjnotificationsmobilenumber.xml:11]: @ankush-maherwal is sql file name changed to match version?

Comment (ankush-maherwal) [reply src/plugins/user/tjnotificationsmobilenumber/tjnotificationsmobilenumber.xml:11]: yes, its updated

**Pull request merged**